projects/js-packages/publicize-components/src/types.ts

```diff
@@ -54,4 +54,5 @@ export interface SocialScriptData {
 	store_initial_state: SocialStoreState;
 	supported_services: Array< ConnectionService >;
 	urls: SocialUrls;
+	is_woa_site: boolean;
```

Comment:
After making that host data change below, we can add the `host` property to [`AdminSiteData` interface](https://github.com/Automattic/jetpack/blob/2b94898223dd1a64e0b6d722bcf9b03908f13b8d/projects/js-packages/script-data/src/types.ts#L18). 

```ts
host: 'woa' | 'atomic' | 'newspack' | 'vip' | 'wpcom' | 'unknown';
```